### PR TITLE
[doc] fix grammar

### PR DESCRIPTION
**Pull request by l2dy**

### CODE REVIEW

**Comment by kou** [comment]
Thanks for reporting this.

I'm not a native English speaker. So I don't know the different of "has been released" and "was released".

I often see "XXX has been released" announce. Why should we use "was released" here?

File: doc/index.html.en [hunk 1]
```diff
@@ -5,7 +5,7 @@ lang_suffix: true
 no_sidebar: true
 ---
 <div class="latest-release">
-  <p><a href="/en/news.html">{{ site.version }}</a> is the latest release. It had been released at {{ site.release_date }}.</p>
```
Comment (l2dy) [inline doc/index.html.en]: > Thanks for reporting this.
> 
> I'm not a native English speaker. So I don't know the different of "has been released" and "was released".
> 
> I often see "XXX has been released" announce. Why should we use "was released" here?

Then I think we agree on that "had been released" is wrong ;)

**Comment by l2dy** [comment]
Regarding the difference, "has been released" is often used without a specific date.

File: doc/index.html.en (hunk 1 above)
Comment (kou) [reply doc/index.html.en]: Ah, "had" is wrong. It's a typo.

**Comment by kou** [comment]
Thanks. I understand.

**Pull request merged**